### PR TITLE
Ncnews 10

### PR DESCRIPTION
**Pull request by jaykeHarrison**
add 201 test and code
add 400 tests: article_id wrong format, body missing properties, body wrong format
add 404 tests: article by ID not found, user by username not found
add documentation to readme

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -249,7 +249,87 @@ describe("PATCH /api/articles/:article_id", () => {
...
+      .send(commentObject)
+      .expect(201)
+      .then(({ body: { addedComment } }) => {
+        expect(addedComment).toEqual(
+          expect.objectContaining({
+            body: "what a wonderful test",
+            votes: 0,
+            author: "rogersop",
+            article_id: 2,
+            created_at: expect.any(String),
```
Comment (scar1377) [inline __tests__/app.test.js:271]: comment_id should be included

File: models/articles.model.js [hunk 2]
```diff
@@ -66,3 +67,30 @@ exports.updateArticleById = (articleId, increaseVotesBy) => {
...
+  return db
+    .query(
+      `
+      INSERT INTO comments (body, votes, author, article_id, created_at)
+      VALUES ($1, 0, $2, $3, $4)
+      RETURNING *;`,
+      [body, username, articleId, createdAt]
+    )
+    .then(({ rows }) => {
+      return rows[0];
```
Comment (scar1377) [inline models/articles.model.js:93]: in seed.js, created_at has a default value NOW(), don't need to be insert in the model

**Review by scar1377 — COMMENTED** [review]
Looks pretty good. Just a couple of things need to change

File: __tests__/app.test.js (hunk 1 above)
Comment (jaykeHarrison) [reply __tests__/app.test.js:271]: ah yes, of course!

File: models/articles.model.js (hunk 2 above)
Comment (jaykeHarrison) [reply models/articles.model.js:93]: done!

**Pull request merged**